aten/src/ATen/ParallelCommon.cpp

```diff
@@ -89,12 +89,19 @@ std::string get_parallel_info() {
 }
 
```

Comment:
nit: could you add some info into get_parallel_info, e.g. whether we're using a mobile thread pool

Reply:
sure, will append "[mobile]" to aten parallel backend string for mobile build.